src/main/java/com/google/sps/data/UrlRequest.java

```diff
@@ -0,0 +1,65 @@
+package com.google.sps.data;
+
+import java.io.IOException;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.UnsupportedEncodingException;
+import java.nio.charset.StandardCharsets;
+import java.net.URLEncoder;
+import java.net.URL;
+import java.net.HttpURLConnection;
+import java.util.Map;
+
+/* Class conatining helper methods to make url requests */
+public final class UrlRequest {
+  /**
+   * Returns the a json string with the API response given an URL path and the query parameters.
+   */
+  public static String urlQuery(String basePath, Map<String, String> parameters) throws IOException {
+    String path = String.format("%s?%s", basePath, getParamsString(parameters));
+    URL url = new URL(path);
+    HttpURLConnection connection = (HttpURLConnection) url.openConnection();
+    connection.setRequestMethod("GET");
+    connection.setRequestProperty("Accept", "application/json");
+    connection.setRequestProperty("Content-Type", "application/json");
+    int responseCode = connection.getResponseCode();
```

Comment:
I don't think you use this anywhere right now.  What is the purpose of fetching the response code?

Reply:
I was planning on adding some exception handler in case the response code is not 200. I'll add a TODO comment.